### DISCUSSION TITLE
posts/nju-program-analysis-a1/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#16] [Announcements]
# posts/nju-program-analysis-a1/

南大软分实验笔记｜A1 存活变量分析与迭代求解器

https://blog.wohin.me/posts/nju-program-analysis-a1/

**Comment by 75ACOL** [comment]
请问怎么解读这一行：[0@L4] x = m; [k, n, x]的意思

**Reply by brant-ruan** [reply]
[k, n, x]是当前行对应的存活变量，m因为后续不再使用，所以不再存活。
